submissions/description/server_security_misconfiguration/no_rate_limiting_on_form/guidance.md

```diff
@@ -1,5 +1,7 @@
 # Guidance
 
-Provide a 1-2 sentence description - see [CVE Project's Key Details Phrasing Guide](http://cveproject.github.io/docs/content/key-details-phrasing.pdf) for tips. Also provide a step-by-step walkthrough with screenshot on how you exploited the vulnerability. This will speed triage time and result in faster rewards!
+Provide a step-by-step walkthrough with a screenshot on how you exploited the vulnerability. This will speed triage time and result in faster rewards.
 
-Your submission MUST include evidence of the vulnerability and not be theoretical in nature. Attempt to escalate the vulnerability to perform additional actions. If this is possible, provide a full proof-of-concept.
+Include a screenshot showing >400 submissions of the form in rapid succession, failing to trigger any rate limiting by the application.
+
+Attempt to escalate the vulnerability to perform additional actions. If this is possible, provide a full Proof of Concept.
```

Comment:
I suggest we include something here that states submissions that rely on IP rotation are not considered to be rate limit bypasses and won't be considered for reward.
(Could be in the template if you feel it makes more sense there)